microservices/tracing-api/dev/values.yaml

```
@@ -11,7 +11,7 @@ service:
 configmap:
   ACCEPTED_AUDIENCES: "http://localhost:4001" #TOCHECK
   APPLICATION_NAME: "interop-tracing-api"
-  API_OPERATIONS_BASEURL: "http://tracing-be-tracing-operations:8080" #TODO TRACING OPERATIONS DNS
+  API_OPERATIONS_BASEURL: "http://tracing-be-tracing-operations:8080" #tracing-operations service URL
```

```suggestion
  API_OPERATIONS_BASEURL: "http://tracing-be-tracing-operations.dev:8080" #tracing-operations service URL
```
As a best practice, let's also specify the domain namespace 👀

---

Is this solution correct?
http://tracing-be-tracing-operations.dev.svc.cluster.local:8080